private/buf/buflsp/completion.go

```diff
@@ -612,6 +612,47 @@ func typeReferencesToCompletionItems(
 			}
 		}
 	}
+	var lastImportLine uint32
+	currentImportPaths := map[string]struct{}{}
+	for currentFileImport := range seq.Values(current.ir.Imports()) {
+		l := currentFileImport.Decl.Span().StartLoc().Line
+		if l < 0 || l > math.MaxUint32 {
+			return nil
```

Comment:
Ah this shouldn't return here?

Reply:
https://github.com/bufbuild/buf/pull/4124/commits/1da0f9e72e25000acefe9e41f22077807b8508e6

Reply:
> Also does the iterator over imports not iterate in with line number increasing?

I guess it does; I wasn't sure if we ought to rely on that behavior in practice, though. I didn't know if a `seq.Indexer` ordering could be relied on.